_posts/2012-05-02-new-utah-political-districts-and-voting-precincts-gis-datasets.md

```diff
@@ -15,7 +15,7 @@ categories:
 ### New Utah Political Districts
 {: .text-left}
 
-[Political districts for state-level offices]({% link data/political/2012-2022-political-districts/index.html %}) (Congressional, State House, State Senate, and State Board of Education) are available as a GIS data layers. These districts are for elections in 2012 and ongoing representation after the new representatives are installed on Jan 1, 2013.
+[Political districts for state-level offices]({% link data/political/index.html %}) (Congressional, State House, State Senate, and State Board of Education) are available as a GIS data layers. These districts are for elections in 2012 and ongoing representation after the new representatives are installed on Jan 1, 2013.
```

Comment:
**question**: is the last sentence of this still accurate? Oh maybe it was in 2012...

**chore**: link to the 2012 shelved data